docs/coa-aks.md

```diff
@@ -0,0 +1,53 @@
+
+### Deployment Models
+
+- ### CoA provisioned AKS account
+    Add the flag "--UseAks true" and the deployer will provision an AKS account and run its containers in AKS rather than provisioning a VM. 
+- ### Shared AKS account with CoA namespace
+    Add the flags "--UseAks true --AksClusterName {existingClusterName}", where the user has admin access to the existing AKS account, the deployer will deploy blob-csi-driver, and aad-pod-identity to the kube-system namespace, and then deploy CoA to the namespace "coa". Add the flag "--AksCoANamespace {namespace}" to override the default namespace.
+- ### Shared AKS account without developer access. 
+    If the user is required to use an AKS account, but does not have the required access, the deployer will produce a Helm chart that can then be installed by an admin or existing CI/CD pipeline. Add the flags "--UseAks true --ManualHelmDeployment". The deployer will print a postgresql command, this would typically be run on the kubernetes node to setup the cromwell user however the user will need to run this manually since the deployer won't directly access the AKS account. 
+
+    - Run the deployer with supplied flags. 
+    - Deployer will create initial resources and pause once it's time to deploy the Helm chart.
+    - Ensure the blob-csi-driver and aad-pod-identity are installed.
+    - Install the CoA Helm chart. 
+    - Run the postgresql command to create the cromwell user. 
+    - Press enter in the deployer console to finish the deployment and run a test workflow. 
+
+### Depedent Kubernetes Packages
+These packages will be deployed into the kube-system namespace.
+- ### Blob CSI Driver - https://github.com/kubernetes-sigs/blob-csi-driver/
+    This is used to mount the storage account to the containers.
+- ### AAD Pod Identity - https://github.com/Azure/aad-pod-identity
+    This is used to assigned managed identities to the containers. 
+
+### External storage accounts
+Typically in CromwellOnAzure you can add storage accounts with input data to the containers-to-mount file. For AKS, you need to modify the values.yaml file of the helm chart and redeploy. The vaules-template.yaml will have examples externalContainers and externalSasContainers. 
+
+### Logs and troubleshooting
+For troubleshooting any of the CoA services, you can login directly to the pods or get logs using the kubectl program. The deployer will write a kubeconfig to the working directory, either copy that file to ~/.kube/config for reference it manually for each command with --kubeconfig {coa-directory}/kubeconfig.txt. You can also run the command `az aks get-credentials --resource-group {coa-resource-group} --name {aks-count} --subscription {subscription-id} --file kubeconfig.txt` to get the file.
```

Comment:
Would suggest "temp" directory instead of working directory; working directory might be someone's downloads folder etc, and we don't want to clutter that up imho

Reply:
Done